### ISSUE TITLE
move followable rolls into a text file

### ISSUE THREAD

**Issue by cameronpenner** [cameronpenner/DiscordBot#11]
RIght now, rolls that users can follow are hard code. These could be moved into a text file so they can be updated without having to change the bot.

**Comment by davidgoering** [comment]
I think I'd rather have this be done automatically. 
You can easily get a list of the server roles. And then we could just remove admin/mod from that list and we should be good to go?

**Comment by cameronpenner** [comment]
yeah, that makes sense. What if, when a user asks to follow a role, the bot makes sure that roll has no permissions and if so, it'll give it to them? Then we'd never need a list at all.
